packages/styles/scss/components/text-area/_text-area.scss

```diff
@@ -29,7 +29,7 @@
     @include type-style('body-01');
     @include focus-outline('reset');
 
-    padding: convert.to-rem(11px) layout.density('padding-inline');
+    padding: convert.to-rem(12px) layout.density('padding-inline');
```

Comment:
Just double checking this update was intentional?

Reply:
@tay1orjones , I reverted this change, but just wanted to check if there was a specific reason for adding it.

Reply:
Yeah this was originally failing the minimum target size rule and the 1px seemed to be a defect when I looked more closely at it. We can remove if it passes without it 👍